### ISSUE TITLE
Some questions about the setting of parameters

### ISSUE THREAD

**Issue by Ma-Zijing** [di0002ya/RFNet#4]
Hi, Ding. I am trying to reproduce RFNet with the code, but I meet some questions. In the config.py, could you give some examples of parameters such as use_para and additional_linear? Since there is no defaults, it is hard to know what kind of values I need to set.

**Comment by Ma-Zijing** [comment]
Also, I am really confused for several points. 
1. It seems RFNet selects training set, validation set and test set fixedly. The codes show that RFNet selects the first 60 environments as the training set, the next 20 environments as validation set, and the last 20 environments as test set. It is weird since I consider the dataset should be seperated randomly, or the performance will be high even though we repeat the experiments several times.
2. In the _data_loader.py_, the function _sample_new_batch. why the variable shots_idx is derived from data_pack.shape[1]? It seems the shape[2] is the number of samples of each gesture. Do I misunderstand? 
I am looking forward to your reply, thanks!

**Comment by bigMaxM** [comment]
Moreover, I wonder why the size of _choose_samples_ in the function of __sample_new_batch_ in the file _data_loader.py_ is k_shots+1？Does it mean RFNet uses k_shots+1 samples of each class to train the model?
